scissors/scissors.c

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <time.h>
+
+#define END_GAME_MESSAGE(user, pc, msg)	\
+	do { printf("Your choise - %s. Computer choise - %s. %s\n",	\
+			user, pc, msg);	\
+			} while(0)
+
+enum errors {
+	NO_ARGS = 1,
+	TO_MUCH_ARGS,	/* 2 */
+	WRONG_ARG,		/* 3 */
+};
+
+int main(int argc, char const *argv[]) {
+	int ret = 0;
+
+	if (argc < 2) {
+		fprintf(stderr, "Need one more argument\n");
+		ret = NO_ARGS;
+		goto end;
+	}
+
+	if (argc > 2) {
+		fprintf(stderr, "To match arguments\n");
+		ret = TO_MUCH_ARGS;
+		goto end;
+	}
+
+	char user_choise[2];
+
+	if ((strcmp(argv[1], "r") == 0) ||
+		(strcmp(argv[1], "p") == 0) ||
+		(strcmp(argv[1], "s") == 0)) {
+
+		strcpy(user_choise, argv[1]);
+	} else {
+		fprintf(stderr, "Argument can be only 'r' 'p' 's'\n");
+		ret = WRONG_ARG;
+		goto end;
+	}
+
+	char game_elements[3][2] = {"r", "p", "s"};

```

Comment:
Is this array const?
Consider making single-dimensional array of char instead of two-dimensional array of strings.